### ISSUE TITLE
Intellisense stopped working when I tried to use node_modules version instead of CDN

### ISSUE THREAD

**Issue by nanachi-code** [suren-atoyan/monaco-react#523]
Everything works fine until I tried to setup the loader to use the node_modules version.
```tsx
import * as monaco from "monaco-editor"
// both didnt work
loader.config({ monaco })
loader.config({
  paths: {
    vs: "node_modules/monaco-editor/min/vs"
  }
})

self.MonacoEnvironment = {
  getWorker(_, label) {
    console.log(label) //? this returns editorWorkerService

    if (label === "typescript" || label === "javascript") {
      return new tsWorker()
    }
    return new editorWorker()
  }
}

// ...
// no intellisense
 <MonacoEditor
          language="javascript" // here I specified to use js only
          path="script.js"
/>
```
I'm using monaco-editor 0.41.0, @monaco-editor/react 4.5.2, vite 4.4.4. This is on a tauri app if that's relevant.

**Comment by suren-atoyan** [comment]
do you see any erros/warning in the console?

**Comment by nanachi-code** [comment]
nope. I suspect there might be something going on with the worker that made it not be able to load because when I used the cdn ver it did log javascript and typescript to the console.

**Comment by nanachi-code** [comment]
somehow when I downgraded to v0.38.0 (same version with the cdn) this works.

> I also tried to move the whole monaco-editor folder to public and config the loader
> 
> ```ts
> loader.config({
>   paths: {
>     vs: "/monaco-editor/min/vs"
>   }
> })
> ```
> 
> but the result was the same. The language worker did not work.

**Comment by DominicH247** [comment]
Also experiencing the same issue. Downgrading monaco-editor to 0.40.0 restores intellisense

**Comment by elisherer** [comment]
My experience is that the editor doesn't recognize "json" as the **first language that is requested** (loading only the base worker, and not jsonMode).
A weird workaround I found was to create an editor setting it to a language other than "json" and creating the one with the "json" language and it loads jsonMode and the worker as expected)

**Comment by ldstein** [comment]
> **Update**: this was resolved in monaco-editor @ 0.42 See: [microsoft/monaco-editor#4159]

Thanks for the pointer. Upgrading `monaco-editor` to latest (0.43.0) resurrected intellisense for me.
